gazebo_ros2_control/src/gazebo_ros2_control_plugin.cpp

```diff
@@ -196,6 +196,18 @@ void GazeboRosControlPlugin::Load(gazebo::physics::ModelPtr parent, sdf::Element
       impl_->model_nh_->get_logger(), "No parameter file provided. Configuration might be wrong");
   }
 
+  std::string controllerManagerNodeName{"controller_manager"};
+
+  std::string controllerManagerPrefixNodeName =
+    sdf->Get<std::string>("controller_manager_prefix_node_name");
+  if (!controllerManagerPrefixNodeName.empty()) {
+    controllerManagerNodeName = controllerManagerPrefixNodeName + "_" + controllerManagerNodeName;
+  }
+
+    std::string controllerManagerNodeNameNamespace{""};
+    controllerManagerNodeNameNamespace = sdf->Get<std::string>("controller_manager_namespace");
```

Comment:
```suggestion
  std::string controllerManagerNodeNameNamespace{""};
  controllerManagerNodeNameNamespace = sdf->Get<std::string>("controller_manager_namespace");
```
we should probably set up a clang-format file here too